metadata.json

```diff
@@ -65,13 +60,18 @@
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "12.04",
         "14.04",
         "16.04",
         "18.04",
         "20.04"
       ]
     },
+    {
+      "operatingsystem": "SLES",
```

Comment:
It's worth to add Support for OpenSuse Leap, too?

Reply:
Leap is the one I am using to test. I am using version 15.2. My understanding is that this is how puppet works for Suse and version here is referring to Leap. The rolling Tumbleweed does not have a fixed version.

Reply:
I talking abut the SLES here, not the version 15. Looking at RHEL, there is a separat metadata section for CentOS, too. I don't know if it's necessary to mention SLES and OpenSuse here, too.

Reply:
Hm. If I looked on https://github.com/puppetlabs/puppetlabs-ntp/blob/main/metadata.json, there is only OpenSuse, just. SLES. I think thats the way to go.